### ISSUE TITLE
Suggestions for vocabularies to do: waterBody, island and island group and sampleSizeUnit and organismQuantityType

### ISSUE THREAD

**Issue by ManonGros** [gbif/vocabulary#145]
Thanks Cecilie for your presentation!

I think that it would be great to have a vocabulary for waterBody (to enable discovery and use of freshwater and marine data).
Similarly, if there could be vocabularies for island and island group, the community might appreciate.

One (difficult) vocabulary that would be great to have would be units! Especially sampleSizeUnit and organismQuantityType. It would facilitate interoperability of the data provided. I don't know how doable this is but that would help a lot in analysis.

**Comment by ymgan** [comment]
Hey, I don't know if there is any plan in the future to have vocabulary also for the measurementUnit of emof extension. [OBIS recommends using emof for samplingSizeUnit](https://manual.obis.org/darwin_core.html#darwin-core-dwc-terms) and BODC vocab [P06](https://vocab.nerc.ac.uk/search_nvs/P06/) collection is the one we use for measurement unit and IDs. If GBIF decides to have vocab for units later on, perhaps the P06 collection can be helpful! 

Cheers

**Comment by ManonGros** [comment]
Suggestion of Vocabulary for waterBody from the draft of Freshwater Guide: "Recommended best practice is to use a controlled vocabulary such as the Getty Thesaurus of Geographic Names."
See also: https://github.com/gbif/doc-freshwater-data-publishing-guide/issues/17

**Comment by CecSve** [comment]
waterBody relevant vocabulary: http://vocab.nerc.ac.uk/collection/C19/current/

**Comment by rubenpp7** [comment]
[Marine regions](https://www.marineregions.org/gazetteer.php?p=search) offers controlled terms for islands, water masses (waterBody?), archipelagos (island groups) and many more oceanic and coastal objects.

They also have some freshwater objects such as rivers and lakes although I'm not sure that how well maintained the freshwater part is since they are dedicated to the Marine realm mainly

**Comment by sformel-usgs** [comment]
For `waterBody`, `island` and `island group` consider also mapping in the GCMD keywords. These are used by NOAA and NASA, who both are OBIS and GBIF contributors. 

Term browser: https://gcmd.earthdata.nasa.gov/KeywordViewer/

1. `waterBody` could start at Oceans and drill down into the terms below it (both json and rdf are available): https://gcmd.earthdata.nasa.gov/kms/concept/ff03e9fc-9882-4a5e-ad0b-830d8f1186cb?format=json

2. Terms relevant to `island` and `island group` are confusing in GCMD. Some are listed under [Oceans](https://gcmd.earthdata.nasa.gov/kms/concept/ff03e9fc-9882-4a5e-ad0b-830d8f1186cb?format=json), like [Canary Islands](https://gcmd.earthdata.nasa.gov/kms/concept/69534766-c309-4145-8963-2bdb7b57340b?format=json), and others under [Continents](https://gcmd.earthdata.nasa.gov/kms/concept/0a672f19-dad5-4114-819a-2eb55bdbb56a?format=json), like [Prince Edward Island](https://gcmd.earthdata.nasa.gov/kms/concept/8d1ef0a1-f9e1-4807-9025-717af90479cb?format=json). We could perhaps work with the GCMD team to make sure we identify them all.

**Comment by ben-norton** [comment]
Two notes.
1. The GCDM is a great thesaurus, but it has a few problems. You describe one of them. The hierarchy:
Continent > North America > Canada > Prince Edward Island doesn't make a lot of sense. Central America and the United States of America are siblings of Canada, which doesn't make a lot of sense either. It's just not consistent. Placing Chukchi Sea under Arctic Ocean makes thematic sense (both are bodies of water, one encapsulating the other), but grouping the Canary Islands and the Gulf of Mexico means the only shared characteristic is geographic location, which isn't very meaningful.
2. Water bodies are tough. EnvThes puts ocean under marine ecosystem with a good definition: https://vocabs.lter-europe.net/envthes/en/page/21811 using a different approach than the GCDM. AgroVoc places oceans as a type of basin (https://agrovoc.fao.org/browse/agrovoc/en/page/c_f6b144da), which I think is rather clever. The GEMET has a good definition, but has a structure where things are grouped into "superclasses". Here, river, lake, and ocean belong to the Hydrosphere group. All of these are much more meaningful than the GCDM.

**Comment by sformel-usgs** [comment]
@ben-norton I completely agree that GCMD shouldn't be the primary vocabulary. I just wanted to point out that it would be valuable to map those terms as part of this controlled vocabulary because we know that some contributors will be using GCMD terms.

**Comment by CecSve** [comment]
Thanks @sformel-usgs and @ben-norton for providing links to sources that might be valuable to reconcile.
